src/main/resources/en_US.properties

```diff
@@ -0,0 +1,202 @@
+# region ERRORS
```

Comment:
I think instead of separating with regions we can create different files on a "en_US" folder